qa-ibd.planx-pla.net/etlMapping.yaml

```diff
@@ -68,3 +68,21 @@ mappings:
           - name: year
             src: year
             fn: set
+  - name: study_viewer_index
+    doc_type: study_viewer_file
```

Comment:
```suggestion
  - name: qa-ibd_project
    doc_type: project
```

Reply:
`qa-ibd_project` to match the convention of `<environment>_<node>` (all QA/Dev envs share an ES server, so we need to be careful when naming indices)